app/Domain/Integrations/Controllers/OrganizerController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations\Controllers;
+
+use App\Domain\Integrations\FormRequests\GetOrganizersRequest;
+use App\Http\Controllers\Controller;
+use App\Search\Sapi3\SearchService;
+use CultuurNet\SearchV3\ValueObjects\Organizer;
+use Illuminate\Http\JsonResponse;
+
+final class OrganizerController extends Controller
+{
+    public function __construct(private readonly SearchService $searchService)
+    {
+    }
+
+    public function getOrganizers(GetOrganizersRequest $request): JsonResponse
```

Comment:
Would perhaps call this class and method `OrganizersController::index` to be closer to the CRUD conventions we used for the other controllers. Or `search` perhaps? Not sure